src/android/NetworkManager.java

```diff
@@ -289,46 +263,41 @@ private void sendUpdate(String type) {
      * @return the type of mobile network we are on
      */
     private String getType(NetworkInfo info) {
-        if (info != null) {
-            String type = info.getTypeName().toLowerCase(Locale.US);
+        String type = info.getTypeName().toLowerCase(Locale.US);
```

Comment:
Can `getTypeName()` ever return null? We've either need to guard this or reverse all `.equals` checks.

Reply:
Seems like it never returns null.